test/blocks/media/media.test.js

```diff
@@ -28,6 +28,10 @@ describe('media', () => {
       const blueButton = actionArea.querySelector('.con-button.blue');
       expect(blueButton).to.exist;
     });
+    it('has a cta container', () => {
+      const ctaArea = medias[0].querySelector('.cta-container .action-area');
+      expect(ctaArea).to.exist;
```

Comment:
Nit: We can turn this into a one liner by not caching ctaArea.
You can just do: `expect(medias[0].querySelector('.cta-container .action-area')).to.exist;`

Reply:
In general I agree but I think it improves readability of the tests so I'll keep them as is.